v2/internal/reconcilers/arm/azure_generic_arm_reconciler_instance.go

```diff
@@ -475,24 +477,44 @@ func (r *azureDeploymentReconcilerInstance) updateStatus(ctx context.Context) er
 // saveAssociatedKubernetesResources retrieves Kubernetes resources to create and saves them to Kubernetes.
 // If there are no resources to save this method is a no-op.
 func (r *azureDeploymentReconcilerInstance) saveAssociatedKubernetesResources(ctx context.Context) error {
+	// Check if this resource has a handcrafted extension for exporting
 	retriever := extensions.CreateKubernetesExporter(ctx, r.Extension, r.ARMClient, r.Log)
 	resources, err := retriever(r.Obj)
-	//_, err := retriever(r.Obj)
 	if err != nil {
 		return err
```

Comment:
Do we need to wrap this error to indicate it was generated by extension code?

Reply:
Done, and for below.